cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -264,6 +264,22 @@
     </description>
   </property>
 
+  <property>
+    <name>twill.security.cconf.secret.disk.name</name>
+    <value>cdap-security</value>
+    <description>
+      The name of the Twill security disk mount which contains cdap-security.xml.
+    </description>
+  </property>
+
+  <property>
+    <name>twill.security.cconf.secret.disk.path</name>
+    <value>.cache</value>
```

Comment:
use `${local.data.dir}/.secret` instead.

Reply:
Whoops, forgot to set this. I was planning to set this to /etc/cdap/security for backward compatibility reasons (since the cdap-security secret disk is typically mounted at that path). Will that work?

Reply:
yes